services/polly.cfc

```diff
@@ -0,0 +1,146 @@
+component {
+
+	variables.service = 'polly';
+
+    public any function init(
+        required any api,
+        required struct settings
+    ) {
+        variables.api = arguments.api;
+        variables.apiVersion = arguments.settings.apiVersion;
+        variables.defaultLanguageCode = arguments.settings.defaultLanguageCode;
+		variables.defaultEngine = arguments.settings.defaultEngine;
+
+        variables.emptyStringHash = hash( '', 'SHA-256' ).lcase();
+
+        return this;
+    }
+
+	/**
+	 * Get list of available voices for the given language.
+	 *
+	 * @link https://docs.aws.amazon.com/polly/latest/dg/API_DescribeVoices.html
+	 */
+	public any function describeVoices(
+		string Engine = variables.defaultEngine,
+		string NextToken = "",
+		boolean IncludeAdditionalLanguageCodes = false,
+		string LanguageCode = variables.defaultLanguageCode
+	) {
+		var requestSettings = api.resolveRequestSettings( argumentCollection = arguments );
+		var queryParams = {
+			'Engine': Engine,
+			'IncludeAdditionalLanguageCodes': IncludeAdditionalLanguageCodes,
+			'LanguageCode': LanguageCode
+		};
+
+		if (Len(NextToken)) {
+			queryParams['NextToken'] = NextToken;
+		}
+
+		var apiResponse = apiCall(
+			requestSettings,
+			'GET',
+			'/v1/voices',
+			queryParams
+		);
+
+		return deserializeJSON(apiResponse.rawData);
```

Comment:
I would like to return the full apiResponse, so I would prefer this be done as follows:

```cfc
apiResponse[ 'data' ] = deserializeJSON(apiResponse.rawData);
return apiResponse;
```